hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,3 @@
+RELEASE_TYPE: patch
+
+This release adds type hints to the `example` and `seed` decorators.
```

Comment:
Same formatting hints as on the other pull; but note that these ones are re-exported so they should be referenced as 

    the :func:`~hypothesis.example` and :func:`~hypothesis.seed` decorators